e2e/iam/project_api_keys_test.go

```diff
@@ -56,6 +56,19 @@ func TestProjectAPIKeys(t *testing.T) {
 		}
 	})
 
+	t.Run("Assign", func(t *testing.T) {
+		cmd := exec.Command(cliPath, iamEntity,
+			projectEntity,
+			apiKeysEntity,
+			"updates",
```

Comment:
do it with `assign`, `updates` is an alias that is not totally correct with the intent of the command